ome_zarr/writer.py

```diff
@@ -122,6 +122,23 @@ def _validate_plate_rows_columns(
     return validated_list
 
 
+def _validate_datasets(datasets: List[dict]) -> List[Dict]:
+
+    validated_datasets = []
+    if datasets is None or len(datasets) == 0:
+        raise ValueError("Empty datasets list")
+    for dataset in datasets:
+        if isinstance(dataset, str):
+            validated_datasets.append({"path": dataset})
+        elif isinstance(dataset, dict):
+            if not dataset.get("path"):
+                raise ValueError("no 'path' in dataset")
+            validated_datasets.append(dataset)
```

Comment:
Should this also validate `coordinateTransformations`? If so, shouldn't the `fmt.validate_coordinate_transformations()` API be migrated here?